code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -1,6 +1,6 @@
+/mob/living/carbon/human/var/emotecooldown = 0

```

Comment:
Isn't there a specific vomit cooldown already for a human?

Reply:
It's defined in "delayed_vomit()" not "vomit()" itself.
delayed_vomit() also uses three spawn() calls which will probably bug Loh.

Reply:
_eyetwitch_